package-lock.json

```diff
@@ -131,13 +131,13 @@
     "@types/diff-match-patch": {
```

Comment:
There were no changes to package.json. Can you remove these changes?

Reply:
yeah

Reply:
those changes were removed.